apps/studio/components/ui/SchemaSelector.tsx

```diff
@@ -61,7 +62,10 @@ const SchemaSelector = ({
     connectionString: project?.connectionString,
   })
 
-  const schemas = (data ?? []).sort((a, b) => (a.name > b.name ? 0 : -1))
+  const schemas = sortBy(
```

Comment:
nit: we able to just use `sort` instead of `sortBy`? wanna leave as little lodash footprint as possible

Reply:
something like this: (untested)
```
const schemas = (data || [])
.filter((schema) => !excludedSchemas.includes(schema.name))
.sort((a, b) => a.name.localeCompare(b.name))
```